### ISSUE TITLE
[DOCS] note header needed in fetch 

### ISSUE THREAD

**Issue by morenoh149** [facebook/react-native#5584]
https://github.com/facebook/react-native/issues/5222#issuecomment-170239302
https://facebook.github.io/react-native/docs/network.html#usage

I tried to formulate a post with a body. It would not work without a header field.

1) is a header necessary if you have a body?
2) if so then this should be documented.
![screen shot 2016-01-28 at 5 27 24 pm](https://cloud.githubusercontent.com/assets/587438/12660913/8d16cd3e-c5e4-11e5-970e-d4ce2b210923.png)

**Comment by morenoh149** [comment]
won't work:
```
  fetch(apiLoginEndpoint, {
    method: 'POST',
    body: JSON.stringify(loginOptions),
  })
```
works:
```
  fetch(apiLoginEndpoint, {
    method: 'POST',
    headers: {
      'Accept': 'application/json',
      'Content-Type': 'application/json',
    },
    body: JSON.stringify(loginOptions),
  })
```

**Comment by shercoder** [comment]
Ran into same issue on Android as well, Adding headers fixed the problem.

**Comment by jabm111** [comment]
I'm getting the same error using Android version 4.4.4 (api 19) and trying to make a post request to an https endpoint (http works). Adding/removing headers doesn't help. It works on Android version 5.1.0 and 6.0.1. I'm using RN v0.20.0.

**Comment by dongrenguang** [comment]
I also get the same error. Finally, I replace the "localhost"(or "127.0.0.1") with my IP address, it is resolved.

**Comment by janicduplessis** [comment]
This is fixed in 87c26885a3520cbb661afcc23efb22943438e52d, it will be shipped in 0.23. It updated the fetch polyfill and now posting without content-type header will use `text/plain;charset=UTF-8` if the type of the body is a string instead of making the request fail.

**Comment by janicduplessis** [comment]
@facebook-github-bot close
